airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -240,7 +240,9 @@ def __init__(self,  # pylint: disable=too-many-arguments,too-many-locals
         self.pod_template_file = pod_template_file
         self.name = self._set_name(name)
         self.termination_grace_period = termination_grace_period
-        self.client = None
+        self.client: CoreV1Api = None
+
+        self.pod: k8s.V1Pod = self.create_pod()
```

Comment:
This will run `self.create_pod()` whenever `KubernetesPodOperator` is initialized, let's keep it None and initialize it separately.

Reply:
@kaxil fixed

Reply:
Can you also add a Description to the PR please describing the bug that is being fixed

Reply:
@kaxil added